src/itest/java/org/kiwiproject/consul/BaseIntegrationTest.java

```diff
@@ -26,6 +26,14 @@ public abstract class BaseIntegrationTest {
 
     public static final GenericContainer<?> consulContainer;
 
+    // NOTE:
+    // The following starts a consul container that will persist across all tests
```

Comment:
what about `org.junit.jupiter.api.extension.ExtensionContext.Store.CloseableResource`?

Reply:
I can look into that, as well as using suites. I think suites are what I want, assuming they support before and after hooks.